include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -367,9 +367,7 @@ class format_help : public format_help_base<format_help>
      * \param[in] layout The command line parameters (e.g. padding) for correct printing.
      * \param[in] tab    The position offset (indentation) to start printing at.
      */
-    void print_text(std::string const & text,
-                    console_layout_struct const & layout,
-                    unsigned const tab)
+    void print_text(std::string const & text, unsigned const tab)
```

Comment:
Why no fixed length integer?

Reply:
because I did not want to touch everything that I designed rather bad 2 years ago when I wrote this :grin: